components/CheckDocumentation/connections/ActiveQuery.tsx

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+
+import {
+  CheckDocs,
+  CheckGuidanceProps,
+  CheckTriggerProps,
+  IssueReferenceBackend,
+} from "../../../util/checks";
+
+import PGDocsLink from "../../PGDocsLink";
+import SQL from "../../SQL";
+import { useSmartAnchor } from "../../SmartAnchor";
+
+const ActiveQueryTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  return (
+    <>
+      <p>
+        Detects queries currently running for longer than the specified
+        threshold of{" "}
+        <code>{config.settings["warning_max_query_age_secs"]}</code> seconds and
+        creates an issue with severity "warning". Escalates to "critical" if the
+        query is still running after{" "}
+        <code>{config.settings["critical_max_query_age_secs"]}</code> seconds.
+        Resolves automatically once the query stops running.
+      </p>
+      <p>
+        Ignores queries from backup programs (<code>pg_dump</code>,{" "}
+        <code>Heroku Postgres Backups</code>), as well as any queries that
+        contain the <code>/* pganalyze:no-alert */</code> or{" "}
+        <code>/* pganalyze=no-alert */</code> magic comment.
+      </p>
+    </>
+  );
+};
+
+const ActiveQueryGuidance: React.FunctionComponent<CheckGuidanceProps> = ({
+  urls: { referenceUrl, serverVacuumsUrl, SettingLink },
+  issue,
+}) => {
+  const Link = useSmartAnchor();
```

Comment:
These are going to be hard to diff because they're new to this repo, but this (across all docs components) is the only change other than imports.